mteb/tasks/Retrieval/est/estqa.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+from datasets import load_dataset
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class EstQA(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="EstQA",
+        dataset={
+            "path": "kardosdrur/estonian-qa",
+            "revision": "766c9bc74b70e6bcdd5a7720a9c7556ac10a8b04",
+        },
+        description=(
+            "EstQA is an Estonian question answering dataset based on Wikipedia."
+        ),
+        reference="https://www.semanticscholar.org/paper/Extractive-Question-Answering-for-Estonian-Language-182912IAPM-Alum%C3%A4e/ea4f60ab36cadca059c880678bc4c51e293a85d6?utm_source=direct_link",
+        type="Retrieval",
+        category="s2p",
+        eval_splits=["train", "test"],
+        eval_langs=["est-Latn"],
+        main_score="ndcg_at_10",
+        date=(
+            "2002-08-24",
+            "2021-05-10",
+        ),  # birth of Estonian Wikipedia to publishing the article
+        form=["written"],
+        domains=["Encyclopaedic"],
+        task_subtypes=["Question answering"],
+        license=None,
+        socioeconomic_status="mixed",
+        annotations_creators="human-annotated",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation="""
+@mastersthesis{mastersthesis,
+  author       = {Anu Käver},
+  title        = {Extractive Question Answering for Estonian Language},
+  school       = {Tallinn University of Technology (TalTech)},
+  year         = 2021
+}
+""",
+        n_samples={"train": 482, "test": 121},
```

Comment:
The test set is quite small, which also seems to make it easier compared to the train set. It might be worth combining then to ensure that it is not too easy?

Reply:
Is the split created for the task or from the dataset source? (if it is created fro the task I would just combine them)

Reply:
I think I created it for the task, so we can just combine it if we want to

Reply:
Would I just do that by saying `eval_splits=["train", "test"]` or do I have to do anything more to get it working?

Reply:
I would do it on the HF side.